src/graph.rs

```diff
@@ -720,14 +728,15 @@ impl<T> Graph<T>
     /// ```
     pub fn map<R: Clone + Debug>(&self, fun: impl Fn(&T) -> R) -> Graph<R> {
```

Comment:
Since we don't have a trait bound for `Clone` and `Debug` any longer, the bounds for this type parameter can also be removed i.e. change `<R: Clone + Debug>` to `<R>`.